src/workloads/execution/SinusoidalReadWrites.yml

```diff
@@ -63,49 +84,116 @@ ActorTemplates:
         Active: {^Parameter: {Name: "ActivePhases", Default: [1]}}
         NopInPhasesUpTo: *NumPhases
         PhaseConfig:
-          SleepBefore: 30 seconds
-          Duration: 1 minute
-          Repeat: 6
+          SleepBefore: 15 seconds
+          Repeat: 3
           CollectionCount: *NumColls
           Operations:
           - OperationName: find
             OperationCommand:
-              Filter: { a: { ^RandomInt: { min: 0, max: *NumDocs } } }
+              Filter: {a: {^RandomInt: {min: 0, max: 100000}}}
             OperationMetricsName: ReadDocs
             ThrowOnFailure: false
+
+- TemplateName: UpdatesActorWithSleepTemplate
```

Comment:
Adding this update actor to update indexed documents definitely adds more stress to the system. Though the "eviction worker thread active" doesn't exceed 4, it is consistently at 4, which would imply that they're doing a lot of work? Let me know if you have other suggestions.

With this phase, we are able to see pretty decent improvements in average throughput and latency.